pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -606,3 +609,10 @@ kʲ	voiceless	velar	stop		palatalization:palatalized
 k̚	voiceless	velar	stop		release:unreleased	
 kʷʼ	voiceless	velar	stop	+	labialization:labialized,ejection:ejective	
 ɡ̤	voiced	velar	stop	+	breathiness:breathy	
+d̪l̪	voiced	lateral	affricate			
+ɽr	voiced	retroflex	trill			
+kp	voiceless	labio-velar	stop			
+gb	voiced	labio-velar	stop			
```

Comment:
same with gb as wit kp, I consider these as double articulation clusters.

Reply:
But CLTS definition considers clusters two different sounds in sequence, in fact the Python object is a sibling of the Diphthong one and the name is "from X to Y". It is true that we would be missing a grapheme for the actual cluster, but as we don't use the top bar (which I agree with, it only complicates stuff) we have to choose one and I would go for the consonants with double articulation.

Reply:
Sorry, `kp` as in clts by now is exactly the double articulation. This is a misunderstanding, but we have always handled in this form. The problem is that you read semantics into the from -> to reading. So this is definitely not needed, as it is handled by CLTS already.